### PR TITLE
Readme review

### PR DESCRIPTION
**Pull request by kathyguerron**
Ready to go! :) @nebtrx

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,20 +24,23 @@ ETA notifications implementation with Node.js/Express and Twilio.
...
+1. Install the application dependencies
 
     ```bash
     $ npm install
     ```
 
-1. Edit the sample environments vars file `.env` to match your configuration. Remember to set your MongoDb connection strings for both environments.
+1. Edit the sample environments file
+
+   Edit the `.env` file to match your configuration. Remember to set your MongoDb connection strings for both environments.
```
Comment (acamino) [inline README.md]: Would you mind prefer _MongoDB_ over _MongoDb_? https://www.mongodb.org/

File: README.md [hunk 2]
```diff
@@ -6,11 +6,13 @@ Status](https://travis-ci.org/TwilioDevEd/eta-notifications-node.svg?branch=mast
...
 
-1. This sample application stores data in a [MongoDB](https://www.mongodb.org/) database using [Mongoose](http://mongoosejs.com/). You can download and run MongoDB yourself (OS X, Linux, Windows).
+1. Install MongoDB
 
-   On OS X, maybe the easiest way to get MongoDB running locally is to install via [Homebrew](http://brew.sh/).
+   This application stores data on a [MongoDB](https://www.mongodb.org/) database using [Mongoose](http://mongoosejs.com/). You can download and run MongoDB yourself (OS X, Linux, Windows).
+
+   On OS X, the easiest way to get MongoDB running locally is to install it via [Homebrew](http://brew.sh/).
 
    ```bahs
```
Comment (acamino) [inline README.md]: Typo: bash

Comment (nebtrx) [inline README.md]: spotted

**Comment by nebtrx** [comment]
LGTM

**Comment by acamino** [comment]
Good work! Squash the commits please.

**Pull request merged**